### ISSUE TITLE
Font size used in Understanding/How to Meet links is 7.5pt

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1130]
<img width="865" alt="Screenshot 2020-05-22 at 14 50 40" src="https://user-images.githubusercontent.com/16352274/82675085-3044af80-9c3c-11ea-9907-f0b84290e66e.png">

https://www.w3.org/TR/WCAG21/#target-size

In Chrome/Firefox the "Understanding Target Size" and "How to Meet Target Size" link font-size is x-small = 10px = 7.5pt

Irrespective of conformance this is pretty hard to click on a touch device.

As an aside, from an editorial standpoint:

- it's difficult to figure out from the WCAG 2.5.5 documents whether the "block of text" exception applies here, since there are no block of text examples (it doesn't look like the link is a full sentence, so the wording of the note in understanding doesn't seem to apply)
- it would be better if the understanding document clarified that both width and height need to be at least 44 pixels

**Comment by giacomo-petri** [comment]
The link now has a height of 15px (if on a single line) with enough padding to ensure it meets the minimum target size requirement.

<img width="812" height="200" alt="Screenshot presenting the new layout of the link (15px height and enough padding)" src="https://github.com/user-attachments/assets/2705a128-9040-4774-b535-b4ecaa99dfbc" />

If you believe the concern remains feel free to reopen.

**Comment by r12a** [comment]
Although there is more space between the links (and perhaps a little too much), the text is still very small.  I would recommend increasing the font size.

I also think that slightly lightening the surrounding box line would help the links stand out better.  (Rounded corners would also look nicer to me, though that's not essential.  Current styling for these pages looks very basic and text-oriented.)

**Comment by patrickhlauke** [comment]
While this is of course subjective, I agree that the current style https://www.w3.org/TR/WCAG22/#target-size-minimum looks weirdly unbalanced

> ![screenshot of the target size SC and its doc links boxout](https://github.com/user-attachments/assets/d32e2c34-86e5-46a6-a138-b393b7da5f32)

it uses `font-size: x-small`, an actual `<br>`, and additional flex `gap: .5em;`

Playing around with some possible tweaks ... removing the `<br>`s altogether, changing the `gap: 1em`, setting `font-size: small`, and tweaking the colour of the boxout border to `#aaaaaa` and giving it a subtle background colour of `#ECF2F7`

> ![screenshot of the target size SC and its doc links boxout, with tweaked stylings](https://github.com/user-attachments/assets/2a1588ff-7660-4090-a01e-79d80431eade)

**Comment by patrickhlauke** [comment]
We could even throw in some additional negative margin, like `margin-top: -3em`, to shunt the boxout a bit further up (making it less likely that it butts up, or worse shifts, following content)

> ![screenshot of the tweaked example, with negative top margin moving the box further up](https://github.com/user-attachments/assets/51d66389-3010-4b0c-8b3d-367b699f6b59)

**Comment by r12a** [comment]
@patrickhlauke this looks much better, and is more readable.

**Comment by alastc** [comment]
@iadawn you might want to give this one an eyes-over. Seems like a reasonable improvement, but it's the style of the main WCAG 2.x doc.

**Comment by iadawn** [comment]
Yeah, I agree that it is an improvement. This should probably be a class 1 editorial change according to the process. I will double check with webmaster if it needs a repub of the whole doc or if we can do an update to the CSS in place.

**Comment by iadawn** [comment]
Webmaster can do this as an in place modification to guidelines.css. If someone can raise a PR with the necessary changes I can get an update sorted out.

Thanks!

**Comment by patrickhlauke** [comment]
> Webmaster can do this as an in place modification to guidelines.css. If someone can raise a PR with the necessary changes I can get an update sorted out.
> 
> Thanks!

Roger, roger ... will do later today

Noting that it's not *just* a CSS change, as i'd ideally like to remove the `<br>`s that are in the source (and suspect get generated automagically somewhere). Will hunt down where that is. Worst case, will add extra CSS to neutralise the existing `<br>`s otherwise.

**Comment by patrickhlauke** [comment]
Done https://github.com/w3c/wcag/pull/4590

**Comment by Kbenge5** [comment]
Yes I want to understand how links meet. Url/DNS/IP address?

kbenge
